modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/checkpoint/CheckpointHistory.java

```diff
@@ -49,6 +50,24 @@
  * This directory holds files for checkpoint start and end.
  */
 public class CheckpointHistory {
+    /** The message appears when no one checkpoint was not reserved for cache. */
+    private static final String NOT_RESERVED_WAL_REASON = "If checkpoint WAL history can't be reserved";
```

Comment:
Failed to perform reservation of historical WAL segment file

Reply:
Done.